docs/UserGuide.md

```diff
@@ -218,6 +223,8 @@ Format: `delete INDEX`
     * `INDEX` refers to the index number shown in the displayed applicant list. 
     * `INDEX` **must be a positive integer** e.g. 1, 2, 3, …
     * `INDEX` of 0 or less, or indexes greater than `2147483647` will be treated as invalid commands.
+* If you made a mistake deleting an applicant and would like to rectify it, please use the [**Undo**](#undoing-an-action--undo) command.
```

Comment:
I think its quite unnecessary to include this for everything. Undo is for when mistakes occur, this seems more repetitive

Reply:
If we are linking commands with hyperlinks I think for standardisation sake we should do for everything as well.

Reply:
Agreed. Undo/redo isn't really a main feature either.